prov/verbs/src/verbs_init.c

```diff
@@ -527,6 +547,7 @@ int vrb_find_max_inline(struct ibv_pd *pd, struct ibv_context *context,
 	} while (qp && (max_inline < INT_MAX / 2) && (max_inline *= 2));
 
 	if (rst != 0) {
+backwards_query: ;
```

Comment:
Is the `;` redundant here?

Reply:
no, it is necessary to resolve build errors